src/js/components/Tag/Tag.js

```diff
@@ -66,7 +85,9 @@ const Tag = forwardRef(
         {onRemove && (
           <StyledRemoveButton
             onClick={onRemove}
+            {...themeTagRemoveProps}
             {...removeProps}
```

Comment:
Related to above comment, but `removeProps` was renamed and now no longer exists. Getting an error locally for missing variable.

I'd say we don't even need to renamed `removeProps` anymore, but can delete "themeTagRemoveProps"